### ISSUE TITLE
NPM scripts missing collapse all button - it was there previously

### ISSUE THREAD

**Issue by JanBN** [microsoft/vscode#225844]
Type: <b>Bug</b>

Hello
NPM scripts sections is missing button for coallpse all. It was there before the last insider update. 
Thanks

VS Code version: Code - Insiders 1.93.0-insider (d751e4324d12370e950f8cd031aa8829637ce300, 2024-08-16T05:03:54.591Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 5500U with Radeon Graphics          (12 x 2096)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|39.38GB (27.64GB free)|
|Process Argv|--crash-reporter-id e1547180-a1ab-4989-b217-e05a7c0ab8e3|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (25)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.5.0
nugetpackagemanagergui|ali|2.1.0
theme-verdandi|be5|2.0.1
gitlens|eam|15.3.0
prettier-vscode|esb|11.0.0
vs-code-xml-format|fab|0.1.5
align-by-regex|jan|1.0.4
vscode-simple-icons|Lau|1.16.0
git-graph|mhu|1.30.0
auto-commit-msg|Mic|0.27.0
csdevkit|ms-|1.9.55
csharp|ms-|2.43.16
vscode-dotnet-runtime|ms-|2.1.1
vscode-nuget-gallery|pat|1.2.1
vscode-versionlens|pfl|1.14.2
code-spell-checker|str|3.0.1
gatito-nightly|Tao|1.2.3
ayu|tea|1.0.5
rest-api-client|unj|1.1.5
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.3.1
vscodeintellicode-completions|Vis|2.0.0
insertnums|vol|0.10.0
markdown-all-in-one|yzh|3.6.2
markdown-editor|zaa|0.1.12

(15 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
vsaa593:30376534
py29gd2263:31024238
vscaat:30438846
c4g48928:30535728
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonnoceb:30776497
asynctok:30898717
dsvsc014:30777825
dsvsc015:30821418
pythonregdiag2:30926734
pythonmypyd1:30859725
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
cppperfnew:30980852
pythonait:30973460
bdiig495:31013172
a69g1124:31018687
dvdeprecation:31040973
dwnewjupyter:31046869
newcmakeconfigv2:31071590
nb_pri_only:31057983
nativerepl1:31104042
refactort:31084545
pythonrstrctxt:31093868
flighttreat:31119334
wkspc-onlycs-c:31111717
nativeloc1:31118317
wkspc-ranged-c:31118571
cf971741:31111988
fcdif617:31111928

```

</details>

![image](https://github.com/user-attachments/assets/36f6bcb7-a49a-49f4-a3f8-84d9175554ca)

**Comment by gjsjohnmurray** [comment]
I am seeing this too, and I have noticed similar occurrences of contributed buttons vanishing from the Source Control view in the current Insiders. By chance I discovered I can make them reappear by dragging the view onto a different container (e.g. Panel). The missing buttons then appear at the new location. Now I can right-click on the view title and choose `Reset Location`. The buttons remain displaying.
